scania/corporate-ui-site
# Feature - tab link url

**Describe pull-request**  
Added functionality to be able to copy URL within tab in a page


**How to test**  
1. Run corporate-ui-site
2. Check all pages are functioning
3. Notice URL when you click in a tab

**Screenshots**  
![image](https://user-images.githubusercontent.com/1199101/85552854-b67d5880-b623-11ea-9759-c88f61471ce6.png)


**Additional context**  
_Add any other context about the pull-request here._
